apiserver/facades/client/client/status.go

```diff
@@ -527,8 +550,8 @@ func fetchAllApplicationsAndUnits(
 	return appMap, unitMap, latestCharms, nil
 }
 
-// fetchRemoteApplications returns a map from application name to remote application.
-func fetchRemoteApplications(st Backend) (map[string]*state.RemoteApplication, error) {
+// fetchConsumerRemoteApplications returns a map from application name to remote application.
+func fetchConsumerRemoteApplications(st Backend) (map[string]*state.RemoteApplication, error) {
```

Comment:
Both offering and consuming models have remote application entities. But we don't show these in the offering model as they are meaningless to the user. They are just the glue and have made up names.